CHANGES.rst

```diff
@@ -1,6 +1,21 @@
 Changelog
 =========
 
+Version 2.0.0
+-------------
+
+Shivashis Padhi created the fundament for a collaboration server in the
+MSS project in google summer of code 2019.
+Tanish Grover continued this work in the google summer of code 2020 and
+overworked our user interface. In addition to the mswms server the project now
+has a mscolab server for editing a flight path.
+
+Vaibhav Mehra has integrated an editor for the MSS setting file.
+
+The KML docking widget was improved in google summer of code 2020 by Aryan Gupta
+
```

Comment:
I think you and Joern should be marked supervisors for these changes, the problem statement is your brainchild :)

But this is a non-blocking suggestion, and thus approving these changes.

Reply:
agreed. also honor the mentors :)